### DISCUSSION TITLE
Delete request not working

### DISCUSSION THREAD

**Discussion by haideralidevs** [activeadmin/activeadmin#7904] [Help]
### Record not deleting

Hey folks, I have using activeadmin (2.12.0) and Rails 6.0.6.1 versions respectively, everything is working fine but delete request is behaving unexpectedly. The issue is as I clicked on delete button, instead of deleting a record it redirects me to the show page, in short it perform `GET` request but it must be a DELETE request as you can see in the below logs

`Started GET "/admin/medical_histories/1" for ::1 at 2023-03-20 02:12:53 +0500
Processing by Admin::MedicalHistoriesController#show as HTML
  Parameters: {"id"=>"1"}
  AdminUser Load (0.4ms)  SELECT "admin_users".* FROM "admin_users" WHERE "admin_users"."id" = $1 ORDER BY "admin_users"."id" ASC LIMIT $2  [["id", 1], ["LIMIT", 1]]
  MedicalHistory Load (0.3ms)  SELECT "medical_histories".* FROM "medical_histories" WHERE "medical_histories"."id" = $1 LIMIT $2  [["id", 1], ["LIMIT", 1]]
  Rendering /home/haider/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/activeadmin-2.12.0/app/views/active_admin/resource/show.html.arb
[Webpacker] Everything's up-to-date. Nothing to do
  Rendered /home/haider/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/activeadmin-2.12.0/app/views/active_admin/resource/show.html.arb (Duration: 13.0ms | Allocations: 8942)
Completed 200 OK in 19ms (Views: 13.6ms | ActiveRecord: 0.8ms | Allocations: 12549)`

I have seen already reported issues but nothing works for me as they are also very ambiguous answers for me,
some of them I checked that `Rack::MethodOverride` is on the place and added jquery in the `active_admin.js` but still same issue.

Here is my `rake middleware`, you can see `use Rack::MethodOverride` exists there but it still its giving me same behaviour.

> use Webpacker::DevServerProxy
> use Rack::Cors
> use ActionDispatch::HostAuthorization
> use Rack::Sendfile
> use ActionDispatch::Static
> use ActionDispatch::Executor
> use ActiveSupport::Cache::Strategy::LocalCache::Middleware
> use Rack::Runtime
> use Rack::MethodOverride
> use ActionDispatch::RequestId
> use ActionDispatch::RemoteIp
> use Sprockets::Rails::QuietAssets
> use Rails::Rack::Logger
> use ActionDispatch::ShowExceptions
> use WebConsole::Middleware
> use ActionDispatch::DebugExceptions
> use ActionDispatch::ActionableExceptions
> use ActionDispatch::Reloader
> use ActionDispatch::Callbacks
> use ActiveRecord::Migration::CheckPending
> use ActionDispatch::Cookies
> use ActionDispatch::Session::CookieStore
> use ActionDispatch::Flash
> use ActionDispatch::ContentSecurityPolicy::Middleware
> use Rack::Head
> use Rack::ConditionalGet
> use Rack::ETag
> use Rack::TempfileReaper
> use Warden::Manager
> use Rack::MethodOverride
> run MedicalSystemRails::Application.routes

and it is my `packs/active_admin.js`

> // Load Active Admin's styles into Webpacker,
> // see `active_admin.scss` for customization.
> import '../stylesheets/active_admin';
> import '@fortawesome/fontawesome-free/css/all.css';
> import 'arctic_admin';
> 
> //= require jquery
> //= require jquery_ui
> //= require jquery_ujs

### Expected behavior
My expected behaviour is that when I perform a DELETE type request it must redirects to the `destroy` action instead of `show` action

**Comment by haideralidevs** [comment]
Hi ActiveAdmin Community,
Please suggest me an answer I'll be thankful to you, as I really stuck into it from two days ago but still on same issue. :pray:

**Reply by matiassalles99** [reply]
Same over here

**Comment by rileys** [comment]
You probably want to take a look at this discussion, sounds like the Javascript it not setup correctly:

https://github.com/activeadmin/activeadmin/discussions/7947
